Fix compilation warnings

ghbaccount opened the pull request:
# Description

This PR fixes some compilation warnings about uninitialized variable and simple memory leaks.

## How This Was Tested

Simple smoke test was performed. Test platform booted successfully without asserts, etc...
Redfish attributes were provisioned and updated.

ghbaccount commented:
Hi @changab , I've created a PR to fix original templates https://github.com/DMTF/Redfish-Schema-C-Struct-Generator/pull/7 .
Could you help to review those changes as well? 
Thank you.

igorkulchytskyy commented on RedfishClientPkg/Library/RedfishFeatureUtilityLib/RedfishFeatureUtilityLib.c:
```diff
@@ -764,10 +764,10 @@ ApplyFeatureSettingsVagueType (
           }
         } else {
-          DEBUG ((DEBUG_MANAGEABILITY, "%a: %a.%a %s value is: 0x%x\n", __func__, Schema, Version, ConfigureKeyLang, RedfishValue.Value.Integer));
+          DEBUG ((DEBUG_MANAGEABILITY, "%a, %a.%a %s value is: 0x%lx\n", __func__, Schema, Version, ConfigureKeyLang, RedfishValue.Value.Integer, Status));
```
Same as above. Status does not have a corresponding "%r" in format string.

ghbaccount replied:
My bad. Fixed.

ghbaccount commented on RedfishClientPkg/Library/RedfishFeatureUtilityLib/RedfishFeatureUtilityLib.c:
```diff
@@ -521,7 +521,7 @@ ApplyFeatureSettingsNumericType (
       }
     } else {
-      DEBUG ((DEBUG_ERROR, "%a: %a.%a %s value is: 0x%x\n", __func__, Schema, Version, ConfigureLang, RedfishValue.Value.Integer));
+      DEBUG ((DEBUG_ERROR, "%a, %a.%a %s value is: 0x%lx\n", __func__, Schema, Version, ConfigureLang, RedfishValue.Value.Integer, Status));
```
Fixed

ghbaccount commented:
Patch to ComputerSystem.V1_13_0.c was removed, because the whole file do not satisfy to Uncrustify requirements

nicklela reviewed (approved):
Looks good to me.

changab commented:
> Hi @changab , I've created a PR to fix original templates [DMTF/Redfish-Schema-C-Struct-Generator#7](https://github.com/DMTF/Redfish-Schema-C-Struct-Generator/pull/7) . Could you help to review those changes as well? Thank you.
> 
> Upd: Oh. I see RedfishClientPkg/ConverterLib/src/ComputerSystem/ComputerSystem.V1_13_0/ComputerSystem.V1_13_0.c can not pass Uncrustify check. I can revert the last two commits and wait for commit to Redfish-Schema-C-Struct-Generator and schema update. It's quite a minor change. You can use these commits for the reference for [DMTF/Redfish-Schema-C-Struct-Generator#7](https://github.com/DMTF/Redfish-Schema-C-Struct-Generator/pull/7)

yeah, as those auto-generated files were not incorporated with Uncrustify. Nickle and I discussed this before,  I think we can either do the Uncristify on the next version of the converter library/driver or skip the CI on those folders.

changab commented:
@ghbaccount , I think all of the comments were addressed, right?

ghbaccount commented:
> @ghbaccount , I think all of the comments were addressed, right?

Sorry! Somehow I missed this comment.
Yes. I fixed unreferenced Status parameter, and removed a patch that made Uncrustify sad.
